src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt

```diff
@@ -0,0 +1,17 @@
+package com.hibob.academy.employeeFeedback.Service
+
+import com.hibob.academy.employeeFeedback.dao.CreationFeedback
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import com.hibob.academy.employeeFeedback.dao.UserLoggedInDetails
+
+
+class FeedbackCreator(private val feedbackDao: FeedbackDao) {
```

Comment:
I'm not sure how it is in kotlin, but class should have an annotation like @Singleton we use in scala?